app/views/candidate_interface/application_form/show.html.erb

```diff
@@ -80,6 +80,11 @@
       <li class="app-task-list__item">
         <%= render(TaskListItemComponent.new(text: t('page_titles.other_qualification'), completed: @application_form_presenter.other_qualifications_completed?, path: @application_form_presenter.other_qualification_path, show_incomplete: false)) %>
       </li>
+      <% unless @application_form.english_speaking_nationality? %>
```

Comment:
I think you need a feature flag condition here.

Reply:
Of course I do🤦🏽. Thanks, good catch!